examples/notebooks/recsys2021/final_submission_feature_engineering.ipynb

```diff
@@ -68,6 +68,7 @@
     " 'TE_mentioned_bucket_id_retweet_timestamp',\n",
     " 'TE_mentioned_bucket_id_retweet_with_comment_timestamp',\n",
     " 'TE_mentioned_bucket_id_like_timestamp',\n",
+    " 'TE_mentioned_bucket_id_reply_timestamp',\n",
```

Comment:
adding extra feature?

Reply:
Not new feature, that's the one missed before. (Totally missed three features, and I saw you have added two, this is another one.)

Reply:
I see, thanks